.evergreen/resync-specs.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# exit when any command fails
+set -e
+PYMONGO=~/Work/mongo-python-driver
+SPECS=~/Work/specifications
+help (){
+echo "Usage: resync_specs.sh [-bcsp] spec"
+echo "Required arguments:"
+echo " spec     determines which folder the spec tests will be copied from."
+echo "Optional flags:"
+echo " -b  is used to add a string to the blocklist for that next run. Can be used"
+echo "     any number of times on a single command to block multiple patterns."
+echo "     Utilizes ."
```

Comment:
Incomplete thought?

Reply:
Yeah, sorry. I realized there are a few errors like that. I will push a commit soon with fixes.